process/configuration/regions/example_ES_Las_Palmas_2023.yml

```diff
@@ -495,3 +495,9 @@ reporting:
     "Spanish - Spain":
       'author_names': 'Agregue nombres de autores editando los ajustes de la configuración de la región usando un editor de texto.'
 #########
+## Optional Google Earth Engine API
+## Set 'gee' to true and enter your Google Cloud project ID to initate Google Earth Engine API connection and generate the following indicators in your analysis:
+## 1. Large Public Urban Green Space (LPUGS) accessibility and availability
+## 2. Global Urban Heat Vulnerability Index (GUHVI)
+gee: true
```

Comment:
default should be false (our assumption is, this involves more work and so may raise the bar for participation; those who are keen may opt in.  In future once we have validated the workflow with users, ensured it is documented and perhaps streamlined it somewhat, then it might be a default inclusion.  But we'll trial things first)

Reply:
Also, include advise that if set to `true` the alternative launcher should be used (`global-indicators.ee.bat` or `global-indicators.ee.sh`, depending on platform)